### PR TITLE
OpenLB

### PR DESCRIPTION
**Pull request by gyear**
For INC1212437- `OpenLB-1.4-0-GCCcore-10.3.0.eb`

* [x] Assigned to reviewers (usually everyone in apps team)

Default:
* [x] EL8-icelake
* [x] EL8-cascadelake
* [x] EL8-haswell

### CODE REVIEW

File: easybuild/easyconfigs/o/OpenLB/OpenLB-1.4-0-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+homepage = 'http://www.openlb.net'
+description = """The OpenLB project provides a C++ package for the implementation of lattice Boltzmann methods that is 
+general enough to address a vast range of tansport problems, e.g. in computational fluid dynamics."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://www.openlb.net/wp-content/uploads/2020/11/']
+sources = ['olb-1.4r0.tgz']
+patches = []
```
Comment (branfosj) [inline easybuild/easyconfigs/o/OpenLB/OpenLB-1.4-0-GCCcore-10.3.0.eb]:
```suggestion
```

File: easybuild/easyconfigs/o/OpenLB/OpenLB-1.4-0-GCCcore-10.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+builddependencies = [('binutils', '2.36.1')]
+
+files_to_copy = ['build', 'doc', 'examples', 'src']
+
+build_cmd = "make && make samples"
+
+sanity_check_paths = {
+    'files': ['examples/laminar/cylinder2d/cylinder2d'],
+    'dirs': ["."]
```
Comment (branfosj) [inline easybuild/easyconfigs/o/OpenLB/OpenLB-1.4-0-GCCcore-10.3.0.eb]:
```suggestion
    'dirs': []
```

File: easybuild/easyconfigs/o/OpenLB/OpenLB-1.4-0-GCCcore-10.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+}
+
```
Comment (branfosj) [inline easybuild/easyconfigs/o/OpenLB/OpenLB-1.4-0-GCCcore-10.3.0.eb:26]: Would it help anyone using this if we set an environment variable point to a sepcific directory? I.e. I see no `bin` or `lib` directory appearing here.

**Review by branfosj — CHANGES_REQUESTED** [review]

**Review by branfosj — APPROVED** [review]

File: easybuild/easyconfigs/o/OpenLB/OpenLB-1.4-0-GCCcore-10.3.0.eb (hunk 3 above)
Comment (gyear) [reply easybuild/easyconfigs/o/OpenLB/OpenLB-1.4-0-GCCcore-10.3.0.eb:26]: From looking at the examples it does seem to be ok

**Pull request merged**